ts/mixpanelConfig/profileProperties.ts

```diff
@@ -55,6 +55,7 @@ type ProfileProperties = {
   SAVED_PAYMENT_METHOD?: number;
   CGN_STATUS: TrackCgnStatus;
   WELFARE_STATUS: ReadonlyArray<string>;
+  FONT_PREFERENCE?: string;
```

Comment:
We also need to add the computation of such property's value in the `updateMixpanelProfileProperties` method

Reply:
addressed in https://github.com/pagopa/io-app/pull/6744/commits/1f36eabdcfb59d1ef16e48a18954f7ba4498abe4